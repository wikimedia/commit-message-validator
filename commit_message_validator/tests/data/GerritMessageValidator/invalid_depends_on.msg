This Depends-On is not valid

Depends-On: pizza
Change-Id: I9cd485c1923dd40f05bd61ef776cb6a9a57187f0
