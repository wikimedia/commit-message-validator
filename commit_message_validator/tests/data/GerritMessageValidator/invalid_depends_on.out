commit-message-validator
Using GerritMessageValidator to check the commit message
The following errors were found:
Line 3: Depends-On: value must be a single Gerrit change id
Please review <https://www.mediawiki.org/wiki/Gerrit/Commit_message_guidelines>
and update your commit message accordingly
